.github/workflows/build_deploy_website.yaml

```diff
@@ -17,16 +17,12 @@ jobs:
     steps:
       - name: Checkout Repo
         uses: actions/checkout@v2
-      - name: Install Node.js
-        uses: actions/setup-node@v2
-        with:
-          node-version: "16"
```

Comment:
Removed as Github runner already has 16 Node installed according to https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2204-Readme.md